go/oasis-test-runner/oasis/args.go

```diff
@@ -482,3 +521,29 @@ func (args *argBuilder) byzantineActivationEpoch(epoch epochtime.EpochTime) *arg
 func newArgBuilder() *argBuilder {
 	return &argBuilder{}
 }
+
+// GetTestPushLabels generates standard Prometheus push labels for the given test.
+//
+// These labels are usually pushed by oasis-test-runner itself or passed to oasis-node workers.
+func GetTestPushLabels(ti env.TestInstanceInfo) map[string]string {
```

Comment:
Why is this defined in `args.go`? It seems like it would be better to have this (and all metrics-related constants) in something like `metrics.go` and only have `appendNodeMetrics` here?